Touch Bar Simulator/TouchBarWindow.swift

```diff
@@ -28,10 +28,31 @@ final class TouchBarWindow: NSPanel {
 				defaults[.lastFloatingPosition] = frame.origin
 			}
 
-			docking?.dock(window: self)
```

Comment:
What if you change `var docking: Docking? {` to `var docking: Docking = .floating {`? I think that could simplify a lot of code. I know you're using it to track certain behavior, it might be simpler to use a separate property for that.